packages/runtime/src/components/shared/Link/index.tsx

```diff
@@ -124,10 +124,15 @@ export const Link = forwardRef<HTMLAnchorElement, Props>(function Link(
 
   if (useNextLink && href != null) {
     return (
-      <NextLink href={href}>
-        {/* eslint-disable-next-line */}
-        <a {...restOfProps} ref={ref} target={target} onClick={handleClick} />
-      </NextLink>
+      <NextLink
+        {...restOfProps}
+        ref={ref}
+        target={target}
+        onClick={handleClick}
+        href={href}
+        // Next.js v12 has legacyBehavior set to true by default
+        legacyBehavior={false}
```

Comment:
So it's defaulted to `true` in Next v12 and we're hardcoding it to `false`, won't it be using the new behavior with Next v13?

Reply:
Oh, I was wrong yesterday when I said we're using the `legacyBehavior`. We remove the `<a>` tag inside the `next/link`, so we're using the new behavior now for both Next.js v12 and Next.js v13.